package.json

```diff
@@ -100,7 +101,10 @@
     "@changesets/cli": "^2.13.1",
     "@commitlint/cli": "16.2.1",
     "@commitlint/config-conventional": "16.2.1",
-    "@emotion/babel-preset-css-prop": "^10.0.14",
+    "@emotion/babel-preset-css-prop": "^11.2.0",
```

Comment:
`@emotion/babel-preset-css-prop` summary of changes:
- added `css-prop` [declaration file to our tsconfig](https://github.com/twilio-labs/paste/pull/2376/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R32).
- [Check out the Emotion v11 upgrade summary](https://emotion.sh/docs/emotion-11#types-overhaul) for more information.